standards_lab/api/views.py

```diff
@@ -63,7 +63,9 @@ def inner(self, request, *args, **kwargs):
 
 
 class ProjectConfig(View):
-    """GET returns the project config and POST updates the config"""
+    """GET returns the project config and POST updates the config.
+    DELETE (maybe confusingly?) does not delete the project config but instead deletes a file from the project.
+    """
```

Comment:
Is there a way we can phrase this you don't think is a judgment opinion?
Because I do think it's worth being clear in the comment that something is not what a lot of people would expect.
Maybe: 

    DELETE does NOT delete the project config but instead deletes a file from the project.

Reply:
In documentation writing we are trying to write sentences in a terse and  "[Active voice](https://developers.google.com/tech-writing/one/active-voice)" way. Framing things around what a function does not do isn't correct for documentation. I've found it tricky to get into the habit of writing in a particular way but it really does make for better documentation.

Reply:
I would also suggest that if this is something we end up fixing soon the comment won't live long anyway :-)

Reply:
How about "DELETE deletes a file from the project (note: does NOT delete the whole project)" - leading with what it does, as I agree documentation should more focus on that, but also having the gotcha noted?

Reply:
> How about "DELETE deletes a file from the project (note: does NOT delete the whole project)" - leading with what it does, as I agree documentation should more focus on that, but also having the gotcha noted?

Yep that sounds fine.